### ISSUE TITLE
[SBRP] Generated artifacts have a mixture of line endings

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/source-build#3500]
1. delete ./src/referencePackages/src/microsoft.bcl.asyncinterfaces/1.1.0
2. run ./generate.sh -x -p microsoft.bcl.asyncinterfaces,1.1.0
3. Inspect the line endings used across the generated artifacts

Expected Results:
1. Artifacts would use the same line ending formats
2. line ending format would be the default for the environment (e.g. Linux/Windows).

Actual Results in a Linux environment:
1. csproj - LF
2. nuspec - CRLF
3. netstandard 2.0 code - CRLF
4. netstandard 2.1 code - LF

**Comment by MichaelSimons** [comment]
nuspec - It looks like this is something that should be handled in RewriteNuspec.cs
csproj - There are some hard coded line endings in GenerateProject that should use {Environment.NewLine}
cs - @ViktorHofer - Can you confirm this is an issue within GenAPI?   Do you know why the inconsistency across the various files?

**Comment by ViktorHofer** [comment]
> cs - @ViktorHofer - Can you confirm this is an issue within GenAPI? Do you know why the inconsistency across the various files?

Are you sure this is related to the new CodeAnalysis based GenAPI? Or did these source files already existed before?

**Comment by MichaelSimons** [comment]
> Are you sure this is related to the new CodeAnalysis based GenAPI? Or did these source files already existed before?

Yes, it try it yourself on a new package in a Linux environment - e.g. ./generate.sh -x -p microsoft.bcl.hashcode,1.1.1

**Comment by ViktorHofer** [comment]
Unfortunately I don't have cycles right now to look into this. Tagging a few more people: @carlossanlop @ericstj @andriipatsula
